package.json

```diff
@@ -1,156 +1,144 @@
-    "urljoin": "0.1.5",
-    "uuid": "2.0.1"
-  },
-  "engines": {
-    "node": ">=4.2.6",
-    "npm": ">=3.6.0"
-  },
-  "devDependencies": {
-    "aws-sdk": "2.1.35",
-    "babel": "6.5.2",
-    "babel-cli": "6.9.0",
-    "babel-core": "6.9.0",
-    "babel-eslint": "6.0.4",
-    "babel-loader": "6.2.4",
-    "babel-plugin-transform-class-properties": "6.9.0",
-    "babel-plugin-transform-runtime": "6.9.0",
-    "babel-polyfill": "6.9.0",
-    "babel-preset-es2015": "6.9.0",
-    "babel-preset-es2015-loose": "7.0.0",
-    "babel-preset-react": "6.5.0",
-    "babel-preset-stage-2": "6.5.0",
-    "babel-runtime": "6.9.0",
-    "body-parser": "1.14.1",
-    "chai": "2.3.0",
-    "chai-as-promised": "5.0.0",
-    "chai-things": "0.2.0",
-    "css-loader": "0.23.0",
-    "deep-assign": "2.0.0",
-    "enquire.js": "2.1.6",
-    "esformatter": "0.9.2",
-    "esformatter-braces": "1.2.1",
-    "esformatter-dot-notation": "1.3.1",
-    "esformatter-jsx": "4.1.4",
-    "esformatter-quotes": "1.0.3",
-    "eslint": "2.4.0",
-    "eslint-plugin-mocha": "4.0.0",
-    "eslint-plugin-react": "4.2.3",
-    "estraverse-fb": "1.3.1",
-    "expose-loader": "0.7.1",
-    "express": "4.13.3",
-    "file-loader": "0.8.5",
-    "grunt": "0.4.5",
-    "grunt-cloudfront": "github:smooch/grunt-cloudfront",
-    "grunt-concurrent": "1.0.0",
-    "grunt-contrib-clean": "0.6.0",
-    "grunt-contrib-concat": "0.5.0",
-    "grunt-exec": "0.4.6",
-    "grunt-gitinfo": "0.1.7",
-    "grunt-release": "jugarrit/grunt-release#master",
-    "grunt-s3": "0.2.0-alpha.3",
-    "imports-loader": "0.6.5",
-    "json-loader": "0.5.4",
-    "karma": "0.13.22",
-    "karma-chai": "0.1.0",
-    "karma-chrome-launcher": "0.1.12",
-    "karma-junit-reporter": "0.2.2",
-    "karma-mocha": "0.1.10",
-    "karma-phantomjs-launcher": "1.0.0",
-    "karma-sinon-chai": "0.3.0",
-    "karma-sourcemap-loader": "0.3.6",
-    "karma-spec-reporter": "0.0.23",
-    "karma-webpack": "1.7.0",
-    "less": "2.5.3",
-    "less-loader": "4.0.3",
-    "matchdep": "0.3.0",
-    "mocha": "2.2.5",
-    "nodemon": "1.9.2",
-    "phantomjs-prebuilt": "2.1.4",
-    "raw-loader": "0.5.1",
-    "react": "15.1.0",
-    "react-addons-test-utils": "15.1.0",
-    "react-dom": "15.1.0",
-    "react-hot-loader": "1.3.0",
-    "redux-mock-store": "1.0.2",
-    "semver": "4.3.6",
-    "sinon": "2.0.0-pre",
-    "sinon-as-promised": "4.0.0",
-    "sinon-chai": "2.7.0",
-    "sinon-stub-promise": "0.1.0",
-    "stats-webpack-plugin": "0.2.2",
-    "style-loader": "0.13.0",
-    "underscore": "1.8.3",
-    "url-loader": "0.5.7",
-    "webpack": "2.6.0",
-    "webpack-cli": "1.0.0",
-    "webpack-dev-middleware": "1.9.0",
-    "webpack-hot-middleware": "2.18.0"
-  },
-  "peerDependencies": {
-    "babel-polyfill": ">=6.9.0 <7.0",
-    "babel-runtime": ">=6.9.0 <7.0",
-    "react": "15.x.x",
-    "react-dom": "15.x.x"
-  },
-  "browser": {
-    "faye": "faye/browser/faye-browser"
-  }
+    "name": "smooch",
```

Comment:
I believe NPM auto-formats your package.json to 2 space indent. Unless that's changed, can we restore it to 2 space indent to prevent it from always going back and forth?